plugins/omprog/omprog.c

```diff
@@ -621,9 +537,10 @@ readPipe(wrkrInstanceData_t *pWrkrData)
 		ABORT_FINALIZE(RS_RET_SUSPENDED);
 	}
 
-	iRet = lineToStatusCode(pWrkrData, line);
-	free(line);
+	CHKiRet(lineToStatusCode(pWrkrData, line));
```

Comment:
This code will change again in my next PR (didn't know at that moment that I would need to later refactor it again). Besides, I don't think it will raise a warning since this pattern (CHKiRet as the last sentence) is used in other places.

Reply:
changed during merge

Reply:
> Besides, I don't think it will raise a warning since this pattern (CHKiRet as the last sentence) is used in other places.

It does, I remember to have changed for more than one occurrence to make Covertiy happy. Maybe it does not detect all other places. It would be interesting if you could pinpoint some so that I can check.